pkg/client/restclient/request.go

```diff
@@ -624,7 +624,7 @@ func (r *Request) tryThrottle() {
 		r.throttle.Accept()
 	}
 	if latency := time.Since(now); latency > longThrottleLatency {
-		glog.Warningf("Throttling request took %v, request: %s:%s", latency, r.verb, r.URL().String())
+		glog.V(2).Infof("Throttling request took %v, request: %s:%s", latency, r.verb, r.URL().String())
```

Comment:
Make this V(4)? I think it was unintentionally a Warningf to begin with, and we really just want it in e2e clusters